modules/sdk/gradle-plugins-maven-plugin-builder/src/main/java/com/liferay/gradle/plugins/maven/plugin/builder/BuildPluginDescriptorTask.java

```diff
@@ -78,6 +78,7 @@ public BuildPluginDescriptorTask() {
 			Conf2ScopeMappingContainer.COMPILE);
 		_configurationScopeMappings.put(
 			"provided", Conf2ScopeMappingContainer.PROVIDED);
+		_pomRepositories.put("provided", Conf2ScopeMappingContainer.PROVIDED);
```

Comment:
Why this? The `_pomRepositories` map is supposed to contain pairs of _repository ID - repository URL_, right?